pipeline-scripts/podvertical.groovy

```diff
@@ -97,7 +98,8 @@ stage ('podvertical_scale_test') {
 						[$class: 'StringParameterValue', name: 'ES_HOST', value: es_host ],
 						[$class: 'StringParameterValue', name: 'ES_PORT', value: es_port ],
 						[$class: 'StringParameterValue', name: 'ES_INDEX_PREFIX', value: es_index_prefix ],
-						[$class: 'StringParameterValue', name: 'EXPECTED_PODVERTICAL_DURATION', value: expected_podvertical_duration ]]
+						[$class: 'StringParameterValue', name: 'EXPECTED_PODVERTICAL_DURATION', value: expected_podvertical_duration ],
```

Comment:
The jjb files of the jobs needs to be modified to add the PROFILE_CLUSTER_OPS option - https://github.com/openshift-scale/scale-ci-pipeline/blob/master/jjb/dynamic/scale-ci_mastervertical.yml.